src/NuGet.Core/NuGet.Packaging/Rules/ContentFolderInPackageReferenceProjectRule.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace NuGet.Packaging.Rules
 {
-    class ContentFolderInPackageReferenceProjectRule : IPackageRule
+    internal class ContentFolderInPackageReferenceProjectRule : IPackageRule
```

Comment:
As per [doc](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers#class-and-struct-accessibility), `Internal` is default access modifier for Classes and structs that are declared directly within a namespace.

Reply:
[Rule #5](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md)